.rubocop.yml

```diff
@@ -26,3 +27,6 @@ Metrics/PerceivedComplexity:
   Max: 10
 Metrics/AbcSize:
   Max: 30
+# https://github.com/rubocop-hq/rubocop/issues/4198
```

Comment:
could you please sync the `.rubocop.yml` with the other baselines (eg. `linux-baseline`)

Reply:
Hi @schurzi,

when I use the .rubocop.yml from linux-baseline we get some style hints:

`flkr@rdde00n6:~/git/postgres-baseline(master)$ rubocop 
warning: parser/current is loading parser/ruby27, which recognizes
warning: 2.7.2-compliant syntax, but you are running 2.7.0.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
/home/flkr/git/postgres-baseline/.rubocop.yml: Warning: no department given for Documentation.
/home/flkr/git/postgres-baseline/.rubocop.yml: Warning: no department given for ParameterAlignment.
/home/flkr/git/postgres-baseline/.rubocop.yml: Warning: no department given for HashSyntax.
/home/flkr/git/postgres-baseline/.rubocop.yml: Warning: no department given for LineLength.
/home/flkr/git/postgres-baseline/.rubocop.yml: Warning: no department given for EmptyLinesAroundBlockBody.
/home/flkr/git/postgres-baseline/.rubocop.yml: Warning: no department given for MethodLength.
/home/flkr/git/postgres-baseline/.rubocop.yml: Warning: no department given for NumericLiterals.
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that can also opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable

Layout/SpaceBeforeBrackets: # (new in 1.7)
  Enabled: true
Lint/AmbiguousAssignment: # (new in 1.7)
  Enabled: true
Lint/DeprecatedConstants: # (new in 1.8)
  Enabled: true
Lint/DuplicateBranch: # (new in 1.3)
  Enabled: true
Lint/DuplicateRegexpCharacterClassElement: # (new in 1.1)
  Enabled: true
Lint/EmptyBlock: # (new in 1.1)
  Enabled: true
Lint/EmptyClass: # (new in 1.3)
  Enabled: true
Lint/LambdaWithoutLiteralBlock: # (new in 1.8)
  Enabled: true
Lint/NoReturnInBeginEndBlocks: # (new in 1.2)
  Enabled: true
Lint/RedundantDirGlobSort: # (new in 1.8)
  Enabled: true
Lint/ToEnumArguments: # (new in 1.1)
  Enabled: true
Lint/UnexpectedBlockArity: # (new in 1.5)
  Enabled: true
Lint/UnmodifiedReduceAccumulator: # (new in 1.1)
  Enabled: true
Style/ArgumentsForwarding: # (new in 1.1)
  Enabled: true
Style/CollectionCompact: # (new in 1.2)
  Enabled: true
Style/DocumentDynamicEvalDefinition: # (new in 1.1)
  Enabled: true
Style/EndlessMethod: # (new in 1.8)
  Enabled: true
Style/HashExcept: # (new in 1.7)
  Enabled: true
Style/NegatedIfElseCondition: # (new in 1.2)
  Enabled: true
Style/NilLambda: # (new in 1.3)
  Enabled: true
Style/RedundantArgument: # (new in 1.4)
  Enabled: true
Style/SwapValues: # (new in 1.1)
  Enabled: true
For more information: https://docs.rubocop.org/rubocop/versioning.html
Inspecting 3 files
.CC`


Imho it is better we rework all rubocup configurations?

Reply:
sure, we can further improve the rubocop configuration. The checked in `rubocop.yml` was already used to lint all other baselines, so I thought this would be a good starting point. Also we need to be careful around some cops, because not everything is compatible with RSpec syntax and we also have a dedicated Inspec Styleguide (https://docs.chef.io/inspec/style) which should be respected.